lib/Runner/run_klee.cpp

```diff
@@ -1575,6 +1580,45 @@ static int run_klee_on_function(
   return 0;
 }
 
+void kill_child(pid_t child_pid) {
+  if (kill(child_pid, 0) != 0) {
+    return;
+  }
+  int statusSIGTERM = kill(child_pid, SIGTERM);
+  std::this_thread::sleep_for(std::chrono::milliseconds(10));
+  if (statusSIGTERM != 0) {
+    int statusSIGKILL = kill(child_pid, SIGKILL);
+    if (statusSIGKILL != 0) {
+      klee_error("Kill with signal SIGKILL return nonzero code.");
+    }
+  }
+}
+
+void wait_until_somebody_dies(const std::vector<pid_t> &child_processes) {
```

Comment:
Name is too bloodthirsty, change to smth like `wait_until_any_fork_dies` or `wait_until_somebody_end`